tests/test/faaslet/test_python.cpp

```diff
@@ -68,26 +121,18 @@ TEST_CASE("Test python state write/ read", "[faaslet]")
     writeCall.set_pythonuser("python");
     writeCall.set_pythonfunction("state_test_write");
     writeCall.set_ispython(true);
-    faaslet::Faaslet faaslet = execFunction(writeCall);
+    execFunction(writeCall);
```

Comment:
I've simplified the `execFunction` interface as it was adding unnecessary complexity to just execute a single function and check it doesn't fail